docs/en/contributor/Building-Alluxio-From-Source.md

```diff
@@ -17,6 +17,24 @@ This guide describes how to clone the Alluxio repository, compile the source cod
 - [Maven 3.3.9 or later](http://maven.apache.org/download.cgi)
 - [Git](https://git-scm.org/downloads)
 
+Alternatively, we have publishhed a docker image [alluxio/alluxio-maven](https://hub.docker.com/r/alluxio/alluxio-maven) with Java, Maven and Git installed to build Alluxio source code.
+For example, to build Alluxio with JDK8, checkout this image at tag `0.0.5-jdk8`:
```

Comment:
to make this easier, i updated the alluxio-maven docker image to have a latest tag pointing to 0.0.5-jdk8
so you can update this to simply checking out the image, which defaults to using the latest tag